BoxContentSDK/BoxContentSDK/BOXContentSDK.h

```diff
@@ -35,7 +35,6 @@
 #import "BOXOAuth2Session.h"
 #import "BOXParallelOAuth2Session.h"
 #import "BOXAppUserSession.h"
-#import "BOXURLSessionManager.h"
```

Comment:
What if one wants to extend the SDK with new functionality? Might knowledge the session manager be needed?